README.md

```diff
@@ -23,6 +23,8 @@ All you need is an empty directory to get started! But if you prefer a starter p
 
 First, install Slinkity + the latest 11ty into your project repo like so:
 
+> Slinkity requires Node 14+. So check `node -v` shows a major version of 14 or higher before installing slinkity.
```

Comment:
Slightly cleaner way to say this:

> Slinkity requires Node 14+. You can check whether you are on a major version of Node 14 by running `node -v` in your terminal before installing Slinkity.

General stylistic question for, @Holben888, do we want to always capitalize Slinkity?

Reply:
Perhaps adding at least as well? Or are we wanting to say this only works with Node 14?

> Slinkity requires Node 14+. You can check whether you are on at least a major version of Node 14 by running node -v in your terminal before installing Slinkity.

The logo and first sentence of the readme used Slinkity in title case. But agree would we should make sure that is the plan going forward 🙂

Reply:
Yeah that's definitely something that might cause some confusion. Are we saying you need to be on 14+ and no higher (as in you shouldn't be on 15 or 16), or just need to be on something higher than 14. I think right now we want users to be on 14 and supporting 16 will require some further tests, but I'll have to double check with @Holben888.

Reply:
Okay sounds like Node 16 should work.

Reply:
@AndrewLeedham @ajcwebdev  I'd say 14 and up to cover our bases! For what it's worth, I got a sample project building and running find on the latest Node (v16.13). Happy to change the wording for clarity:

> Slinkity requires Node v14 and up. You can check your versioning by running `node -v` in your terminal before trying Slinkity.

Reply:
Only just realised I didn't capitalise the second occurrence 😅 Good catch @ajcwebdev.